packages/controllers/src/snaps/SnapController.ts

```diff
@@ -403,6 +416,7 @@ const snapStatusStateMachineConfig = {
           target: SnapStatus.running,
           cond: disabledGuard,
         },
+        [SnapStatusEvent.update]: SnapStatus.installing,
```

Comment:
Would it be better to have a `SnapStatus.updating`? I wonder if there will be a need to distinguish between a first-time install or an update 🤔

Reply:
Specifically decided not have additional state. Don't think it's necessary and it complicates logic

Reply:
I've thought about whether we should have `SnapStatus.updating`, but we _could_ be fine without it, and it's quite nice to not have to worry about it in this PR. We can always complicate the state machine later if needed!